### PR TITLE
Daylight: Overflow-group

### PR DESCRIPTION
**Pull request by GeorgePlukov**

### CODE REVIEW

File: components/overflow-group/README.md [hunk 1]
```diff
@@ -1,10 +1,40 @@
 # Overflow Groups
+The `d2l-overflow-group` element can be used to add responsiveness to a set of buttons, links or menus. 
 
-## d2l-overflow-group
-The `d2l-overflow-group` element can be used to add responsiveness to a set of buttons, links or menus. Items added to this container element will no longer wrap onto a second line when the container becomes to small but will be added too a dropdown menu with configurable styling.
+<!-- docs: demo size:medium autoSize:false display:block -->
```
Comment (margaree) [inline components/overflow-group/README.md:4]: super nit: I've been trying to alphabetize the properties (other than if `name` is there I usually put that first)

File: components/overflow-group/README.md [hunk 2]
```diff
@@ -1,10 +1,40 @@
...
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/overflow-group/overflow-group.js';
```
Comment (margaree) [inline components/overflow-group/README.md:7]: Would this need the other imports? Like dropdown button etc.

File: components/overflow-group/README.md [hunk 3]
```diff
@@ -1,10 +1,40 @@
...
+		</d2l-dropdown-menu>
+	</d2l-dropdown>
+	<d2l-button>Copy</d2l-button>
+	<d2l-button>Import</d2l-button>
+	<d2l-button>Delete</d2l-button>
+</d2l-overflow-group>
+```
+
+## Overflow Group [d2l-overflow-group]
+Items added to this container element will no longer wrap onto a second line when the container becomes to small but will be added too a dropdown menu with configurable styling.
```
Comment (margaree) [inline components/overflow-group/README.md:31]:
```suggestion
Items added to this container element will no longer wrap onto a second line when the container becomes too small, but will be added to a dropdown menu with configurable styling.
```

**Review by margaree — APPROVED** [review]

File: components/overflow-group/README.md (hunk 2 above)
Comment (GeorgePlukov) [reply components/overflow-group/README.md:7]: The buttons/dropdowns that get used are imported within the component so we don't technically need to add them here.

**Pull request merged**